smoke-test/tests/cypress/cypress/e2e/query/query_tab.js

```diff
@@ -1,75 +1,71 @@
 const DATASET_URN = 'urn:li:dataset:(urn:li:dataPlatform:hdfs,SampleCypressHdfsDataset,PROD)';
+const runId = Date.now()
```

Comment:
Will this `runId` be regenerated for each retry of cypress test? If not, it might be a good idea to make this a parameter for the functions and create it within the 3 `it` blocks

Reply:
Yes it will regenerated **runId** for each retry of cypress